docs/_data/tv.csv

```diff
@@ -1,4 +1,5 @@
 name,image,title,designer
+The_Godfather,330,,
```

Comment:
Should this not also be New_Retro_Wave_(band), or possibly the incorrect number has been assigned here?

The 2 images added in the music and tv .csv files are the same image, so I'd assume the names should match here

Reply:
Sorry for that, I added the godfather logo before realising, that it already exists. I created a new commit, which deletes that row.